application/libraries/Email.php

```diff
@@ -10,4 +10,14 @@ public static function send($to, $subject, $message)
 
 			mail($to, $subject, $message, $headers);
 		}
+		
+		public static function receive($from, $subject, $message, $pseudo)
```

Comment:
Par rapport à la méthode `send` juste au-dessus, il n'y a que le header "Reply-to" qui est ajouté. Il devrait y avoir un moyen d'éviter des copiers/collers de lignes :)